rust/tools/authorization-logic/src/souffle/datalog_ir.rs

```diff
@@ -0,0 +1,17 @@
+// This file contains the datalog IR (DLIR) which makes the translation from
```

Comment:
I think this would be better if structured as a doc comment. (see https://doc.rust-lang.org/rust-by-example/meta/doc.html)

Reply:
applied broadly